lib/Email/Folder/Search.pm

```diff
@@ -0,0 +1,111 @@
+package Email::Folder::Search;
+
+# ABSTRACT: wait and fetch search from mailbox file
+
+=head1 NAME
+
+Email::Folder::Search
+
+=head1 DESCRIPTION
+
+Search email from mailbox file. This module is mainly to test that the emails are received or not.
+
+=head1 SYNOPSIS
+
+    use Email::Folder::Search qw(get_email_by_address_subject clear_mailbox);
+    $Email::Folder::Search::mailbox = '/var/spool/mbox';
+    my %msg = get_email_by_address_subject(email => 'hello@test.com', subject => qr/this is a subject/);
+    clear_mailbox();
+
+=cut
+
+=head1 Functions
+
+=cut
+
+use strict;
+use warnings;
+use Email::Folder;
+use Encode qw(decode);
+
+use base qw(Exporter);
+
+our @EXPORT_OK = qw(get_email_by_address_subject clear_mailbox);
+
+our $VERSION = '0.01';
+
+=head2 $mailbox
+
+The path of mailbox file.
+
+=cut
+
+=head2 $timeout
+
+The seconds that get_email_by_address_subject will wait if the email cannot be found.
+
+=cut
+
+# mailbox is set in the chef postfix part and travis-script/setup-postfix
```

Comment:
Since this is a CPAN module, there shouldn't be any reference to Chef or internal repos. I'd suggest this default is also not going to be valid for anyone other than us!